internal/render/pod.go

```diff
@@ -314,6 +318,21 @@ func (*Pod) Statuses(ss []v1.ContainerStatus) (cr, ct, rc int) {
 	return
 }
 
+// LastRestart returns the last container restart time.
+func (*Pod) LastRestart(ss []v1.ContainerStatus) (latest metav1.Time) {
```

Comment:
Don't think this should be exported since it's only used here currently.

Reply:
I followed the same approach that exists on the `TestCheckPodStatus` test, which access the exported func `Statuses`. 

I tried to find some solutions for this, as the `render` package is not on the same `render_test` package, the function would not be accessible from the test file. Do you have any suggestion on how to avoid this issue? Should I create a wrapper to export it? Looks like more of a workaround.

Reply:
@ericbsantana I see. We typically leverage xxx_int_test.go to test un-exported calls.